tests/ui/destructuring-assignment/struct_destructure_fail.stderr

```diff
@@ -17,6 +17,8 @@ error: base expression required after `..`
    |
 LL |     Struct { a, .. };
    |                   ^ add a base expression here
+   |
+   = help: using struct update syntax requires a base expression of the same type as the struct to fill out the remaining fields
```

Comment:
This does not seem helpful here -- we just need to mention that it needs a base expression *at all* :laughing:

Reply:
Maybe this can be made into a suggestion... e.g.

```
Struct { a, ../* expr */ }
              ++++++++++
```

.. using `/* expr */` to signify that something needs to go there.